be/src/storage/storage_engine.cpp

```diff
@@ -543,6 +549,47 @@ void StorageEngine::_start_clean_fd_cache() {
     VLOG(10) << "Cleaned file descriptor cache";
 }
 
+void StorageEngine::compaction_check() {
+    int checker_one_round_sleep_time_s = 1800;
+    while (!bg_worker_stopped()) {
+        MonotonicStopWatch stop_watch;
+        stop_watch.start();
+        LOG(INFO) << "start to check compaction";
```

Comment:
```
LOG(INFO) << "start to check compaction";
```
The log is non-sense.

Reply:
it is useful to get background task info. I tend to keep it.